packages/phone_log/lib/phone_log.dart

```diff
@@ -18,9 +18,9 @@ class PhoneLog {
   PhoneLog.private(MethodChannel platformChannel):_channel = platformChannel;
 
   /// Check a [permission] and return a [Future] with the result
-  Future<bool> checkPermission() async {
-    final bool isGranted = await _channel.invokeMethod("checkPermission", null);
-    return isGranted;
+  Future<PermissionStatus> checkPermission() async {
```

Comment:
This is a breaking change. Please document it in CHANGELOG.md and up the version.

Also update the documentation. "return a Future with the result" is not very informative. Thanks!

Reply:
Done.